trame_server/core.py

```diff
@@ -386,6 +386,7 @@ def start(
         backend: str = "aiohttp",
         exec_mode: str = "main",
         timeout: int = None,
+        host: str = "localhost",
```

Comment:
Maybe this should be `None` and not set on the options if `None` to let `wslink` choose the default better?

Reply:
I think pushing `localhost` is fine... unless we want to use `os.environ.get("TRAME_HOST", "localhost")` instead when None

Reply:
Done. I chose `TRAME_DEFAULT_HOST` to signify its a default that can be overridden